recipes/entrez-direct/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
   build:
     # The installer script downloads statically compiled binaries, so this is effectively just avoiding linter errors
     - {{ compiler('go') }}
+  run:
+    - wget
```

Comment:
Why is wget needed?

Reply:
All of the e-utils depend on either `wget` or `curl`. busybox includes `wget`, but `extended-base` doesn't, so this dependency was hidden previously.